templates/forms/DropdownField.ss

```diff
@@ -1,5 +1,5 @@
 <select $AttributesHTML>
 <% loop $Options %>
-	<option value="$Value.XML"<% if $Selected %> selected="selected"<% end_if %><% if $Disabled %> disabled="disabled"<% end_if %>><% if Title %>$Title.XML<% else %>&nbsp;<% end_if %></option>
```

Comment:
that test is passing for me locally...

![unit-test](https://cloud.githubusercontent.com/assets/878176/8295778/1e3cbfa8-199c-11e5-8812-1dfff26c662d.png)

any ideas?